tests/entry_page_test.js

```diff
@@ -27,131 +26,98 @@ Scenario('Verify confirmations when both experiments matching', async ({ I }) =>
+        allActiveKeysEntryPointInfo: allActiveKeysEntryPointInfo,
+        countOfEntryPoints: countOfEntryPoints,
+        countOfConfirmations: countOfConfirmations
+    }
+
+    return experimentDetails;
+}
+
+Scenario('Verify confirmation when both experiments match the context and the first has entry point equal to true', async ({ I }) => {
+    // Act
+    const page = '/index.html';
+    const exp = await arrangeTest(I, page);
+
+    // Assert
+    I.assertTrue(exp.countOfEntryPoints === exp.countOfConfirmations);
+    I.assertTrue(exp.countOfConfirmations > 0);
+
+    exp.confirmations.forEach(confirmation => {
+        exp.allActiveKeysEntryPointInfo.find(key => {
+            if (key.isEntryPoint) {
+                I.assertTrue(confirmation.indexOf(key.id) > -1);
+            } else {
+                I.assertTrue(confirmation.indexOf(key.id) === -1);
+            }
+        })
     })
-
-    let experimentsConfig = await utils.getConfiguration();
- 
-    const entryPoints = await utils.getEntryPoints(activeKeys, experimentsConfig);
-   
-    //Assert
-    entryPoints.forEach(element => {
-        if(element.isEntryPoint){
-            I.assertContain(confirmations[entryPoints.indexOf(element)], element.id);
-        }else{
-            I.assertNotContain(confirmations[entryPoints.indexOf(element)], element.id);
-        }
-    });
 });
 
-Scenario('Verify no confirmations when none of experiment matching', async ({ I }) => {
-    //Act
-    I.amOnPage('/');
-    I.wait(2);
-    let confirmations = await I.executeScript(async () => {
-        let confirmationsArray = [];
-       
-        if(evolv.context.remoteContext.experiments.confirmations){
-
-            evolv.context.remoteContext.experiments.confirmations.forEach(element => {
-                confirmationsArray.push(element.cid);
-            })
-            return confirmationsArray;
-        }
-        else {
-            return [];
-        };
-    });
-
-    let activeKeys = await I.executeScript(async () => {
-        let keys = await evolv.client.getActiveKeys();
-        let filteredKeys = keys.current.filter(key => key.split('.').length < 3);
-        return filteredKeys.map(key => key.split('.')[1]);
-    })
-
-    let experimentsConfig = await utils.getConfiguration();
- 
-    const entryPoints = await utils.getEntryPoints(activeKeys, experimentsConfig);
-   
-    //Assert
-    entryPoints.forEach(element => {
-        if(element.isEntryPoint){
-            I.assertContain(confirmations[entryPoints.indexOf(element)], element.id);
-        }else{
-            I.assertNotContain(confirmations[entryPoints.indexOf(element)], element.id);
-        }
-    });
-
-    I.assertEqual(confirmations.length, 0);
-    entryPoints.forEach(element => {
-         I.assertEqual(element.isEntryPoint, false);
-    });
-});
+// Scenario('Verify confirmation when both experiments match the context and the second has entry point equal to true', async ({ I }) => {
```

Comment:
Going to merge this, so we can get the regression tests passing again -- but can you bring back the other scenarios with 2 entry points - reusing your extracted function?